src/desktop/preload.js

```diff
@@ -1,27 +1,43 @@
 // @flow
-import {ipcRenderer, remote} from 'electron'
 
 /**
  * preload scripts can only load modules that have previously been loaded
  * in the main thread.
  */
+let remote = require('electron').remote
```

Comment:
Do you need to do `require('electron')` now? You also do it multiple times. I would expect that it compiles to the same thing but maybe I'm wrong.